django_modern_rest/validation.py

```diff
@@ -297,6 +310,15 @@ class ModifyEndpointPayload:
     error_handler: SyncErrorHandlerT | AsyncErrorHandlerT | Empty
     allow_custom_http_methods: bool
 
+    # OpenAPI documentation fields:
+    summary: str | None = None
```

Comment:
Please, use a common base class for common fields. We don't want to duplicate this.